src/main/webapp/src/components/CourseDetails.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div>
+    <div>
+      <h1>{{ course.title }}</h1>
+    </div>
+    <div>
+      <div class="detailThumbnail">
+        <b-img left thumbnail class="mb-3 mr-3" :src="cardimage(course)"></b-img>
+      </div>
+      <b-row class="mb-3" cols="1" cols-md="2">
+        <b-col v-if="course.courseType">
+          <div class="label">Typ:</div>{{ course.courseType=='EXTERNAL' ? 'Externe' : 'Interne' }}
+        </b-col>
+        <b-col v-if="course.location">
+          <div class="label">Ort:</div>{{ course.location=='REMOTE' ? 'Remoteveranstaltung' : 'Präsenzveranstaltung' }}
```

Comment:
Location would be displayed as "Präsenzveranstaltung" if it is undefined/null

Reply:
it would not be displayed because of the v-if above